pallets/parachain-staking/src/lib.rs

```diff
@@ -45,19 +56,34 @@
 //!
 //!
 //! # Aggregation
-//! Aggregation feature allows for delegating under single candidate(A) but with many different liquidity
-//! tokens. At the moment every candidate has some specific liquidity token assigned that can be
-//! used for delegations `[Pallet::join_candidates]`. Single candidate = single liquidity token. If collator/candidate
-//! wants to enable more tokens for people to delegate with that can be done only using aggregation feature:
-//! - new candidacy(B) needs to be submitted (with that particular liq token id) `[Pallet::join_candidates]`
-//! - candidate (B) needs to be enabled/approved by candidate (A) using `[Pallet::aggregator_update_metadata]`
-//! - candidate (B) needs to assign himself to candidate (A) that he wants to delegate under `[Pallet::update_candidate_aggregator]`
-//!
-//! Now people can delegate multiple liquidity tokens to multiple candidates but all on behalf
-//! single collator.
+//! Aggregation feature allows increasing single candidate total stake/valuation(aggregator stake) by assosiating 
+//! other candidates stake with aggregator candidate. Each candidate needs to bond different
+//! liquidity token
+//! ```
+//!                            ####################
+//!               -------------#  Candidate A     #-------------
+//!              |             # token: MGX:KSM   #             |
+//!              |             ####################             |
+//!              |                       |                      |         
+//!              |                       |                      |         
+//!              |                       |                      |         
+//!              |                       |                      |         
+//!              |                       |                      |         
+//!              |                       |                      |         
+//!      ####################  ####################  ####################
+//!      #  Candidate B     #  #  Candidate C     #  #  Candidate D     #
+//!      # token: MGX:TUR   #  # token: MGX:IMBU  #  # token: MGX:MOVR  #
+//!      ####################  ####################  ####################
+//! ```
+//! 
+//! If candidate decides to aggregate under some other candidate it cannot be chosen to be collator,
+//! instead if aggregator will be chosen he will get part of the rewards.
 //!
-//! ** Once candidate decides to aggregate under some other candidate it wont be chosen as collator,
-//! only top level(those who don't aggregate under someone else) candidates can be chosen**
+//!```
+//!                        candidate B MGX valuation
+//! Candidate B rewards = ------------------------------------ * Collator A total staking rewards
+//!                        candidate (A + B + C + D) valuation
+//!```
```

Comment:
A candidate cannot simultaneously also be an aggregator.

"Aggregation feature allows several candidates total stake valuation to be aggregated (summed) under an aggregator."

Here B,C and D cannot aggregate under A. As A is a candidate.